x/evidence/types/evidence_test.go

```diff
@@ -72,18 +72,6 @@ func TestEquivocationValidateBasic(t *testing.T) {
 	}
 }
 
-func TestEvidenceAddressConversion(t *testing.T) {
```

Comment:
Maybe we could comment this test out with a `TODO HV2` ?

Reply:
True, but preferred to remove it as it invokes some functions which are (and won't be) there anymore, like the ones setting the prefix from the config, e.g. `SetBech32PrefixForConsensusNode`
So preferred to remove it as this module is not going to be used anyway.
WDYT?

Reply:
This test seems to verify whether the validator address is calculated correctly from the one provided in the evidence. Now in the near future it can be important once we start "acting" on comet evidences of misbehaviour and even though it's not a holistic test, the assertions being done here are elementary. 
So I think we can leave it commented and tweak the test later to our needs.

Reply:
Addressed [here](d08375894471206ac168791a0c14e60999bc646b)